### DISCUSSION TITLE
Textbox Typing

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#1214] [Ideas] [answered]
I haven't really seen the "textbox" typing anywhere except on obscure keyboard websites. It's simple, instead of typing the words and having them highlight as your typing them your typing inside of a box that's empty. If you get a word incorrect it'll highlight the word inside the box that you got wrong.

**Comment by Miodec** [comment, marked as answer]
Sure.. but why. It would need a rewrite and goes against everything that the website goes for. And its hard to word with textboses

**Reply by frogfrogfrogfrogfrog** [reply]
Simply an idea just to throw in. The fact that it would need a complete rewrite makes me rethink that it wasn't a good idea in all honesty.
